heuristic/heuristic_calculations.py

```diff
@@ -139,7 +141,7 @@ def calculate_f_for_employee(data, e, soft_vars, weights, w):
 
     f = (
         weights["rest"] * sum(
-            min(100, w[e, j][1])
+            min(72, w[e, j][1])
```

Comment:
Remove the min-calculation here if you implement my previous suggestion

Reply:
See my comment above. I believe this is a better way to do it. We should not restrict an employees rest period if they are able to have longer breaks. We should just not reward the longer periods